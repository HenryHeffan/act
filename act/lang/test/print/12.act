pint X = 100;

defproc test()
{
   int w;
   chp {
     w := {0, 5, int(X{3..0},4)};
     log ("%x got:", w)
   }
}

test t;
