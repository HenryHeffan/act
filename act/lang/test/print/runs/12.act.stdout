defproc test ();

defproc test ()
{

/* instances */
int<32> w;

/* connections */
chp {
w:={0x0,0x5,0x4};log("%x got:",w)
}
}


/* instances */
test t;

/* connections */
